.github/workflows/ci_cd.yml

```diff
@@ -17,7 +17,7 @@ env:
   ANSRV_GEO_PORT: 700
   ANSRV_GEO_LICENSE_SERVER: ${{ secrets.LICENSE_SERVER }}
   GEO_CONT_NAME: ans_geo
-  RESET_IMAGE_CACHE: 8
+  RESET_IMAGE_CACHE: 9
```

Comment:
I really need to start looking into this -- the issue is caused by a random order color assignment by matplotlib... and it seems to be changing constantly..